project/Dependencies.scala

```diff
@@ -35,8 +36,9 @@ object Dependencies {
     val pulsarFunctionsApi = "org.apache.pulsar" % "pulsar-functions-api" % V.pulsar
 
     // Testing
-    val munitCore       = "org.scalameta" %% "munit"            % V.munit
-    val munitScalacheck = "org.scalameta" %% "munit-scalacheck" % V.munit
+    val munitCore       = "org.scalameta"       %% "munit"            % V.munit
+    val munitScalacheck = "org.scalameta"       %% "munit-scalacheck" % V.munit
```

Comment:
If we add weaver, do we actually need munit at all?

Reply:
The unit tests still run using MUnit, I have no problems in ditching it.

Reply:
I would be happy to have only one framework, can we easily drop munit once and for all now? Or it would require too much rework? If so, then we could create another ticket and do it separately

Reply:
No, it's pretty easy, I can do it 👍🏾